internal/corazawaf/waf.go

```diff
@@ -276,6 +277,7 @@ func NewWAF() *WAF {
 		RuleEngine:                types.RuleEngineOn,
 		RequestBodyAccess:         false,
 		RequestBodyLimit:          _1gb,
+		RequestBodyLimitAction:    types.BodyLimitActionReject,
```

Comment:
As per https://coraza.io/docs/seclang/directives/#secrequestbodylimitaction, the default value for the request body limit action is `Reject`. We can't rely on the zero value of the BodyLimit action because we want to have two different default values for RequestBodyLimitAction (reject) and ResponseBodyLimitAction (processPartial). The first one has been explicited here